dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/helloworld/HelloWorldTest.java

```diff
@@ -119,36 +126,18 @@ private StringBuilder runGetMethod(int exprc, String requestUri, String testOut)
         }
     }
 
-    private StringBuilder runDeleteMethod(int exprc, String testOut)
+    private int runGetMethodInvalidCharset(String requestUri)
```

Comment:
ignore the comparison, this is a new method that git wants to conflict with another one

Reply:
Maybe you (or your IDE) deleted the old one?  I don't think anything calls it, so it's fine to remove it in any case.

Reply:
ah, you are right, it was unused and I did delete it. Forgot about that.